### ISSUE TITLE
 miss ReLU?

### ISSUE THREAD

**Issue by leedoge** [CoinCheung/BiSeNet#51]
class ConvBNReLU(nn.Module):
    def __init__(self, in_chan, out_chan, ks=3, stride=1, padding=1, *args, **kwargs):
        super(ConvBNReLU, self).__init__()
        self.conv = nn.Conv2d(in_chan,
                out_chan,
                kernel_size = ks,
                stride = stride,
                padding = padding,
                bias = False)
        self.bn = BatchNorm2d(out_chan)
        self.init_weight()

    def forward(self, x):
        x = self.conv(x)
        x = self.bn(x)
        return x

**Comment by CoinCheung** [comment]
No, I used inplace-abn, which binds bn and relu together. Please refer to this [repo](https://github.com/mapillary/inplace_abn).

**Comment by leedoge** [comment]
oh, i see.
thank you very much old iron!
